tests/test_clusters.py

```diff
@@ -32,6 +34,26 @@ def cluster_provisioner_mocks(mocker):
     }
 
 
+def test_cluster_form_defaults(client, test_user, ssh_key):
+    response = client.post(reverse('clusters-new'), {}, follow=True)
+
+    form = response.context['form']
+
+    assert form.errors
+    assert (form.initial['identifier'] ==
+            '{}-telemetry-analysis'.format(user_display(test_user)))
+    assert form.initial['size'] == 1
+    assert form.initial['ssh_key'] == ssh_key.id
+
+
+def test_no_keys_redirect(client, test_user):
+    response = client.post(reverse('clusters-new'), {}, follow=True)
+    assert response.status_code == 200
+    assert response.redirect_chain[-1] == (reverse('keys-new'), 302)
+    assert ('No SSH keys associated to you' in
+            [m for m in get_messages(response.wsgi_request)][0].message)
```

Comment:
I'll test using `request.response` instead. I didn't see a helper listed in the pytest django docs. I agree it's kind of ugly.